bcrypt/bcrypt_test.go

```diff
@@ -241,3 +241,14 @@ func TestNoSideEffectsFromCompare(t *testing.T) {
 		t.Errorf("got=%q want=%q", got, want)
 	}
 }
+
+func ExampleBcrypting() {
+	password := []byte("mypassword")
+	securedPassword, err := GenerateFromPassword(password, DefaultCost)
```

Comment:
Maybe you can rename it `hashedPassword`, like the first param of `CompareHashAndPassword`

Reply:
It's salted and hashed. As `saltedHashedPassword` was a little to long for me, I called it secured.
Does that make sense?

Reply:
Ok. But `securedPassword` comes from nowhere. But it's clear to me.
I try to improve.
But you're right. :+1: